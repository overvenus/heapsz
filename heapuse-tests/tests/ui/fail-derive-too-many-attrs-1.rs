use heapuse_derive::Heap;
use heapuse_example::*;

#[derive(Heap)]
pub struct UnknownAttr {
    #[heap(add, add)]
    pub a: primitive::StructPrimitives,
}

fn main() {}
