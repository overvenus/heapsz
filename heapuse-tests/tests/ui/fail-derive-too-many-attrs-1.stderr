error: too many heap attributes
 --> tests/ui/fail-derive-too-many-attrs-1.rs:6:7
  |
6 |     #[heap(add, add)]
  |       ^^^^^^^^^^^^^^
